lib/vcr/library_hooks/typhoeus.rb

```diff
@@ -117,4 +135,3 @@ def self.vcr_response_from(response)
     WebMock::HttpLibAdapters::TyphoeusAdapter.disable!
   end
 end
-
```

Comment:
I think the convention for VCR is to have a newline at the end of every file, could you re-introduce these lines, please?